code/modules/awaymissions/mission_code/raisearm.dm

```diff
@@ -0,0 +1,443 @@
+			src.say(pick("RAAAAAGGHHHH!!!","AAAARRRGGHHHH!!!","RRRAAUUUGGHH!!!"))
+
+
+/mob/living/simple_animal/hostile/nsoldier/bomber //think serious sam's kamikazes but with mini-hitlers
+	name = "Mini-F�hrer"
+	desc = "A small, robotic recreation of the F�hrer himself, it seems like he wants to tell you something."
+	icon_state = "miniheil"
+	icon_living = "miniheil"
+	speed = 1
+	maxHealth = 25
+	health = 25
+	harm_intent_damage = 5
+	melee_damage_lower = 5
+	melee_damage_upper = 7
+	attacktext = "heils"
+	deathmessage = "The Mini-F�hrer explodes!"
+	loot = list(/obj/effect/gibspawner/robot)
+
+/mob/living/simple_animal/hostile/nsoldier/bomber/AttackingTarget()
+	..()
+	if(iscarbon(target))
+		var/mob/living/carbon/C = target
+		if(prob(90))
+			C.Weaken(2)
+			src.say("HEIL HITLER!")
+			explosion(src, 0, 0, 2, 3, 2)
+			src.gib()
+
+/mob/living/simple_animal/hostile/spawner/heilfactory
+	name = "production factory"
+	icon = 'icons/obj/mining.dmi'
+	icon_state = "bin_full"
+	spawn_text = "marches out of"
+	max_mobs = 3
+	mob_type = /mob/living/simple_animal/hostile/nsoldier/bomber
+	faction = list("german")
+
+/*
+ *Structure-related stuff
+ */
+
+/turf/simulated/floor/plating/marble
+	name = "marble flooring"
+	icon_state = "marble"
+	broken_states = list("marblebroken")
+	burnt_states = list("marblescorched")
+
+
+/obj/effect/mob_spawn/human/corpse/nsoldier
+	name = "Geo-Nazi"
+	uniform =  /obj/item/clothing/under/nsoldier
+	shoes = /obj/item/clothing/shoes/combat
+	gloves = /obj/item/clothing/gloves/combat
+	mask = /obj/item/clothing/mask/gas/old
+	helmet = /obj/item/clothing/head/helmet/stalhelm
+	has_id = 0
+
+
+//space billboards
+
+/obj/structure/billboard
+	name = "Space billboard"
+	desc = "The pinnancle of space-advertisement."
+	icon = 'icons/obj//banters/billboard.dmi'
+	icon_state = "generic"
+	density = 0
+	layer = 10
+	anchored = 1
+
+/obj/structure/billboard/random
+
+/obj/structure/billboard/random/New()
+	..()
+	icon_state = "billboard_[rand(1,5)]"
+
+
+//pillars
+
+/obj/structure/pillar
+	name = "pillar"
+	desc = "A reaaaaaally tall pillar"
+	icon = 'icons/obj/pillar.dmi'
+	icon_state = "pillarbottom"
+	anchored = 1
+	density = 1
+	layer = 10
+
+/obj/structure/pillar/broken
+	name = "broken pillar"
+	desc = "The remains of a pillar."
+	icon = 'icons/obj/pillarbroke.dmi'
+	icon_state = "pillarbroke"
+
+
+//banners
+
+/obj/structure/banner
+	name = "banner"
+	desc = "A piece of cloth hung up on the wall."
+	icon = 'icons/obj/banters/banter.dmi'
+	icon_state = "generic"
+	anchored = 1
+	density = 0
+	layer = 10.1
+
+/obj/structure/banner/heil
+	name = "nazi banner"
+	desc = "A banner depicting a rather rude image. Has a strange, smokey smell to it."
+	icon_state = "heil"
+
+/obj/structure/banner/cyka
+	name = "russian banner"
+	desc = "A banner. Smells faintly of vodka."
+	icon_state = "cyka"
+
+/obj/structure/banner/assblastusa
+	name = "patriotic banner"
+	desc = "A banner depicting a bald eagle."
+	icon_state = "assblastusa"
```

Comment:
*american* banner

Also where's our banner you can't leave the world's uppitiest aircraft carrier out of this